idea/jvm-debugger/jvm-debugger-test/testData/stepping/custom/functionCallStoredToVariable.ir.out

```diff
@@ -0,0 +1,21 @@
+LineBreakpoint created at functionCallStoredToVariable.kt:7
+LineBreakpoint created at functionCallStoredToVariable.kt:14
+LineBreakpoint created at functionCallStoredToVariable.kt:21
+LineBreakpoint created at functionCallStoredToVariable.kt:28
+Run Java
+Connected to the target VM
+functionCallStoredToVariable.kt:7
+functionCallStoredToVariable.kt:4
+functionCallStoredToVariable.kt:14
```

Comment:
This one is interesting. The code is:

```
11: val b = {
12:   // STEP_OVER: 2
13:   // Breakpoint!
14:   1
15: }
```

We are breaking on line 14 and then we take two steps before resuming.

On JVM_IR those two steps are line 11 which is leaving the accessibility bridge, followed by line 43 leaving the caller.

On JVM those two steps are line 43 leaving the caller (this particular bridge has no line numbers) followed by line 11 for the assignment.

So it looks like line 11 and line 43 are reversed, but they are not, there is just an extra line 11 step on JVM_IR because of the consistent line numbers in bridges.